packages/browser-extension/source/util/localizations.ts

```diff
@@ -0,0 +1,244 @@
+
+
+export function getValuesFromLocalizationASTJSON(astLocale:SourceFile | undefined) {
+    let english:string[] = []
+    let slovak:string[] = []
+    let locales:string[] = []
+    let positionsTable:any[] = []
+    astLocale?.forEachChild((child:any)=>{
+      child?.expression?.properties?.forEach((property:any)=>{
+        console.log("Property",property)
+        locales = [...locales, property.name.text]
+        locales = [...locales, property.initializer.text]
+        positionsTable = [...positionsTable, {text:property.initializer.text, position:property.initializer.pos }]
+        english = [...english,property.name.text]
+        slovak = [...slovak, property.initializer.text]
+      })
+    })
+    console.log(positionsTable)
+      return  {
+        english,
+        slovak,
+        locales,
+        positionsTable
+      }
+}
+
+export function createFinalFile(printer:any, finalAST:any){
+    console.log("FINAL AST", finalAST)
+    console.log(printer.printFile(finalAST))
+    console.log(finalAST?.print())
+}
+
+const project = new Project();
+
+const localeSource = project.createSourceFile("skSK.ts", JSON.stringify(skSK))
+
+export function transformSourceFile(positions:any[]) {
+  localeSource.transform((traversal:any) => {
+      const node = traversal.visitChildren();
+      const found = positions.find((position:any)=>position?.position == node.pos)
+      if(found?.position == node.pos && ts.isStringLiteral(node)) {
+        node.text = found.text
+        return ts.factory.createStringLiteral(found.text)
+      }
+        return node
+      })
+      console.log("New Source",localeSource)
+      localeSource.transform((traversal:any) => {
+        const node = traversal.visitChildren();
+          return node
+        })
+      return localeSource
+}
+
+export function getValuesFromLocalizationAST(astLocale:SourceFile | undefined) {
+  let expressions:any = []
+  let locales:string[] = []
+    const found = astLocale?.statements.find(statement=>statement)
+    //@ts-ignore
+    const statements = found?.statements
+    statements.forEach((statement:any)=>{
+      expressions = [...expressions, statement.expression]
+    })
+    expressions.forEach((expression:any)=>{
+      if (expression.text){
+        locales = [...locales, expression.text]
+      } else {
+        locales = [...locales, expression.left.text]
+        locales = [...locales, expression.right.text]
+      }
+    })
+    return locales
+}
+
+export function getLocales(locales:string[]) {
+    let english:string[] = []
+    let slovak:string[] = []
+      
+      locales.forEach((locale:string)=>{
+        if (locales.indexOf(locale) % 2 == 0){
+          english = [...english, locale]
+        } else {
+          slovak = [...slovak, locale]
+        }
+      })
+      return{
+          english,
+          slovak
+      }
+}
+
+export function createTable(english:string[], slovak:string[], panelWindow:Window) {
```

Comment:
DOM related functions should be place in a diffrent file and not mixed with AST related code.